### PR TITLE
[vision] chore: parameterized FROM, add cloudbuild.yaml

### PR DESCRIPTION
**Pull request by tmatsuo**
also update README with how to build and upload the images.

@genquan9 
Now the images are little bit more maintenable. Potentially we can parameterize the version number too.

### CODE REVIEW

File: vision/automl/edge_container_predict/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM tensorflow/serving:1.14.0
+ARG FROM_SUFFIX
```
Comment (genquan9) [inline vision/automl/edge_container_predict/Dockerfile:15]: FROM_SUFFIX to VERSION ?
And take 1.14.0 or 1.14.0-gpu as versions?

File: vision/automl/edge_container_predict/cloudbuild.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 3600s
+
+steps:
+    - name: 'gcr.io/cloud-builders/docker'
+      args: ['build', '-t', 'gcr.io/$PROJECT_ID/gcloud-container-1.14.0', '.']
```
Comment (genquan9) [inline vision/automl/edge_container_predict/cloudbuild.yaml:19]: set VERSION for both cpu and gpu dockers?

File: vision/automl/edge_container_predict/Dockerfile (hunk 1 above)
Comment (tmatsuo) [reply vision/automl/edge_container_predict/Dockerfile:15]: Ah I see.

Other thoughts on this, see:
https://github.com/GoogleCloudPlatform/python-docs-samples/pull/3841

We are using renovate bot which constantly trying to update the versions. Does it make sense to update it to 1.15.0? If so, maybe we should have separate Dockerfiles for cpu and gpu without any ARGs.

WDYT?

Comment (tmatsuo) [reply vision/automl/edge_container_predict/Dockerfile:15]: @genquan9 Also, I think it is better to run the test against the newly built images, and if it succeeds, we can push the new image to gcr.io.

**Comment by tmatsuo** [comment]
@genquan9 PTAL

For a record, if we want to update the tensorflow image version, you can do something like the following:

```bash
gawk -i inplace '{ gsub(/1.14.0/, "1.15.0") }; { print }' *
```

**Comment by tmatsuo** [comment]
The error message: "Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.38"

I'll add an env var to fix it.

**Pull request merged**